neo/Consensus/ConsensusService.cs

```diff
@@ -155,6 +171,16 @@ private void InitializeConsensus(byte view_number)
                 if (context.MyIndex == context.PrimaryIndex)
                 {
                     context.State |= ConsensusState.Primary;
+                    if (!context.State.HasFlag(ConsensusState.SignatureSent))
```

Comment:
Is possible that this state returns `false`?

Reply:
Yes. This will happen after a `changeview` sometimes.

Reply:
But if this is false, how they Fill the TXs?

Reply:
They needn't. The transactions have been filled  before changing view.